specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -59,36 +58,25 @@ export function Notifications({
 
     let timeout: NodeJS.Timeout | undefined = undefined;
 
-    function doFetch(since?: Date): void {
-      const startFetchTimestamp = new Date();
-
-      const now = new Date();
-      const year = now.getFullYear();
-      const month = now.getMonth().toString();
-      const day = now.getDate().toString();
-      const hours = now.getHours().toString();
-      const minutes = now.getMinutes().toString();
-      const seconds = now.getSeconds().toString();
+    function formatDate(date: Date) {
+      const year = date.getFullYear();
+      const month = date.getMonth().toString();
+      const day = date.getDate().toString();
+      const hours = date.getHours().toString();
+      const minutes = date.getMinutes().toString();
+      const seconds = date.getSeconds().toString();
 
-      const formattedDate = `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
-
-      const sinceDate = since;
-      const sinceYear = sinceDate?.getFullYear();
-      const sinceMonth = sinceDate?.getMonth().toString();
-      const sinceDay = sinceDate?.getDate().toString();
-      const sinceHours = sinceDate?.getHours().toString();
-      const sinceMinutes = sinceDate?.getMinutes().toString();
-      const sinceSeconds = sinceDate?.getSeconds().toString();
+      return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
+    }
 
-      const formattedSince = `${sinceYear}-${sinceMonth}-${sinceDay} ${sinceHours}:${sinceMinutes}:${sinceSeconds}`;
+    function doFetch(since?: Date): void {
+      const startFetchTimestamp = new Date();
 
-      const queryString = since ? formattedSince : formattedDate;
+      const sinceValue = since ? formatDate(since) : formatDate(new Date());
```

Comment:
```suggestion
const singeValue = formatDate(since ?? new Date())
```

or replace
```ts
function doFetch(since?: Date): void {
```

with
```ts
function doFetch(since? = new Date()): void {
```
and then this line will be simpler:
```suggestion
const sinceValue = formatDate(since);
```

at which point, if you want, you can even remove the `sinceValue` variable, and just inline the `formatDate(since)` when you call formatUrl()